tests/ci/azure_pipeline_test/dsvm_nightly_linux_cpu.yml

```diff
@@ -31,18 +31,16 @@ jobs:
       python ./scripts/generate_conda_file.py --name nightly_reco_base && \
       conda env create --quiet -f nightly_reco_base.yaml 2> log
     displayName: 'Setup Conda Env'
-    timeoutInMinutes: 10
 
   - script: |
```

Comment:
This task should include a "failOnStandardError: True" so that if the conda install does not work, the task will fail and the build will not continue.